### ISSUE TITLE
PyPy support

### ISSUE THREAD

**Issue by mjatti** [chaimleib/intervaltree#61]
Hi,

I'd like to use intervaltree within a pypy based Docker container. Running docker build with base image python:2.7-14-slim works smoothly, but the same setup with base image pypy:2-5.8-slim exits with "intervaltree-2.1.0-cp27-none-any.whl is not a supported wheel on this platform."

Tests for intervaltree pass with "pypy setup.py test". My host is OSX and PyPy version: Python 2.7.13[pypy-5.8.0-final].

Is there some specific reason for not having pypy listed in setup.py classifiers?

Thanks!

**Comment by chaimleib** [comment]
Hi!

I'm sorry, but I'm not very familiar with Docker and I don't recognize this error. It looks like there is something wrong with my setup.py, but I didn't quite get it. Could you dumb it down for me?

**Comment by chaimleib** [comment]
I figured out what I needed to add. Thanks for making this issue!
